profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/plugins/tmgmt_poetry.plugin.inc

```diff
@@ -469,7 +475,10 @@ xsi:noNamespaceSchemaLocation=\"http://intragate.ec.europa.eu/DGT/poetry_service
 
     // Create soap client.
     try {
-      $client = new SoapClient($poetry['address'], array('cache_wsdl' => WSDL_CACHE_NONE, 'trace' => 1));
+      $client = new SoapClient($poetry['address'], array(
+        'cache_wsdl' => WSDL_CACHE_NONE,
+        'trace' => 1,
```

Comment:
@Fefaine & @voidtek maybe we should consider to remove 'trace' option which is by default FALSE. It seems that it has an impact on the tests performance. To test that I've just removed two occurrences in the following files:
- nexteuropa_dgt_connector/tmgmt_poetry/plugins/tmgmt_poetry.plugin.inc
- nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry_mock/src/Mock/PoetryMock.php

All tests passed without 'trace' option. Maybe follow up ticket :)